app/User.php

```diff
@@ -35,4 +35,8 @@ public function teaches() {
 	public function enrolledIn() {
 		return $this->belongsToMany('App\Course')->join('users as usrs', 'course_user.user_id', '=', 'usrs.id')->where('usrs.role', config('enum.user_roles')['student']);
 	}
+
+	public function teamsLed() {
```

Comment:
Should we be adding one for the teams where I belong?

Reply:
Good idea. No sé si resolver esto aquí o mejor esperarme al user story de Add users to my team (creo que todavía no hay card para ese). Thoughts?

Reply:
Nvm, acabo de hablar esto con @katiearriagam y por datos que se necesitan en front-end es conveniente que se implemente de una vez 👍 .